skymp5-client/src/front/components/actorvalues.ts

```diff
@@ -19,8 +19,15 @@ export const getActorValues = (ac: Actor): ActorValues => {
     return resultActorValue;
 }
 
-export let baseActorValues: Map<string, unknown>; 
+export const setActorValuePercentage = (ac: Actor, avName: string, baseAV: number, percentage: number) =>
```

Comment:
please use Alexey's solution for consistency (with k = 1) https://github.com/skyrim-multiplayer/skymp/commit/243428db32ed55df70d926345b0fff79e5bbda8f
in the future, we could remove code duplication. it would be nice!
btw the `baseAV` argument should not be present.